### DISCUSSION TITLE
Using gt to produce tables in pdf (LaTeX) documents

### DISCUSSION THREAD

**Discussion by HaroldDMiller** [quarto-dev/quarto-cli#6095] [Feature Requests]
### Description

I would really like to have gt work with Quarto via the standard LaTeX engines in order to produce nicely formatted tables.  The other table generation packages, such as kableextra, are vastly inferior to gt in terms of overall capabilities, but they are compatible with Quarto pdfs.  gt and Quarto work beautifully together for html output and produce far superior results to other packages, but they don't work together for pdf output: the font sizes/colors/line widths/etc that are saved as inline css are not carried through to pdfs via LaTeX.  The solution I'm currently using is to output the gt tables as a png file and then display the png file as an image.  This preserves all of the formatting added by gt, but the scaling of the image has to be done manually in order to have some semblance of consistency in font size, but it still generally looks bad if there are multiple tables of different widths.  I don't know whether the best solution is to change gt or Quarto or both, but it seems to me to be important functionality for Quarto to have.

**Comment by mcanouil** [comment]
As I said (https://github.com/quarto-dev/quarto-cli/issues/6084#issuecomment-1617891876), this is not up to Quarto CLI, but up to `gt`.
I am closing this in favour of your other opened issue on the relevant repository.

- rstudio/gt#1363
